src/shared/Git-Credential-Manager/Program.cs

```diff
@@ -20,10 +21,12 @@ public static void Main(string[] args)
             using (var app = new Application(context))
             {
                 // Initialize TRACE2 system
+                // TRACE2 convention is for main thread of execution to be named "main"
+                Thread.CurrentThread.Name = "main";
```

Comment:
Since each helper is a child processes, should all of these `Program.cs` threads not also be called `main`? It's just like Git starting itself as a child, no?

Reply:
I re-organized the below method you suggested a bit to account for this.

Reply:
@ldennington do we still need to explicitly name this thread?
It's the entry thread so will be `ManagedThreadId == 0`, which we call `"main"`.

Reply:
Nope, took care of that!